dist/index.html

```diff
@@ -1,12 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-<script defer src="/Leaderboard/runtime.bundle.js"></script><script defer src="/Leaderboard/main.bundle.js"></script></head>
-<body>
-    <h1>Hello world!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>THE GAME OF THRONES</title>
+    <script defer src="/Game-Of-Thrones/runtime.bundle.js"></script><script defer src="/Game-Of-Thrones/main.bundle.js"></script></head>
+    <body>
+        <main>
+            <header>
+            <h1>THE GAME OF THRONES</h1>
+            <nav>
+              <div class="logo">
+              </div>
+              <ul class="navItem">
+                <li class="list-item">
+                  <a href="#" class="actors">Actors(counter)</a>
+                </li>
+                <li class="list-item">
+                  <a href="#">Movies</a>
+                </li>
+                <li class="list-item">
+                  <a href="#">Series</a>
+                </li>
+              </ul>
+            </nav>
+            </header>
+            <section class="display-container">
+              <div class="character-list" id="character-list">
+
+              </div>
+            </section>
+            <section class="modal" id="commentModal">
+              <div class="modal-content">
+                <span class="close">&times;</span>
+                <p>Some text in the Modal..</p>
+              </div>
+            </section>
+        </main>
+
+   <!-- <script src="bundle.js"></script> -->
```

Comment:
💊 Kindly remove commented code as it makes your codebase untidy

Reply:
Solved